packages/lab/src/slider/RangeSlider.tsx

```diff
@@ -245,6 +245,10 @@ export const RangeSlider = forwardRef<HTMLDivElement, RangeSliderProps>(
           handleKeydownOnThumb={(event) => handleKeydownOnThumb(event, 0)}
           offsetPercentage={`${calculatePercentage(value[0], max, min)}%`}
           trackDragging={isDragging && thumbIndexState === 0}
+          isFocusVisible={isFocusVisible && thumbIndexState === 0}
+          inputRef={inputRefs[0]}
+          onFocus={() => handleFocus(0)}
```

Comment:
as we are assigning a function to a prop, any impact on performance since this change was added ?